pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1384,6 +1395,27 @@ func (n *NetworkPolicyController) getInternalGroupWorkloads(group *antreatypes.G
 	return pods, ees, nil
 }
 
+func (n *NetworkPolicyController) triggerPolicyResyncForLabelIdentityUpdates(key string) {
+	n.internalNetworkPolicyMutex.RLock()
+	klog.V(2).InfoS("Resyncing policy for LabelIdentity events", "policy", key)
+	internalNPObj, found, _ := n.internalNetworkPolicyStore.Get(key)
+	n.internalNetworkPolicyMutex.RUnlock()
+	if !found {
+		return
+	}
+	internalNP := internalNPObj.(*antreatypes.NetworkPolicy)
+	switch internalNP.SourceRef.Type {
+	case controlplane.AntreaClusterNetworkPolicy:
+		if cnp, err := n.cnpLister.Get(internalNP.SourceRef.Name); err == nil {
```

Comment:
Like we didn't check whether the source NetworkPolicy exists after getting internal NetworkPolicy in `syncAppliedToGroup` and `syncAddressGroup`:
https://github.com/antrea-io/antrea/blob/18aee9df8e8293ac984dfed2d6de1c79f74bd6ea/pkg/controller/networkpolicy/networkpolicy_controller.go#L1298-L1310
I think the case that internal NetworkPolicy exists but source NetworkPolicy does not exist is rare, and syncInternalNetworkPolicy could handle it correctly anyway, not querying it here could save both overhead for most cases and code lines.

Reply:
Done